docs/examples/events/on_decorator03.py

```diff
@@ -0,0 +1,38 @@
+from textual import on
+from textual.app import App, ComposeResult
+from textual.widgets import Button
+
+
+class OnDecoratorApp(App):
+    CSS_PATH = "on_decorator.css"
+
+    def compose(self) -> ComposeResult:
+        """Three buttons."""
+        yield Button("Bell", id="bell")
+        yield Button("Toggle dark", classes="toggle dark")
+        yield Button("Quit", id="quit")
+
+    @on(Button.Pressed, button="#bell")  # (1)!
+    def play_bell(self):
+        """Called when the bell button is pressed."""
+        self.bell()
+
+    @on(Button.Pressed, button=".toggle.dark")  # (2)!
+    def toggle_dark(self):
+        """Called when the 'toggle dark' button is pressed."""
+        self.dark = not self.dark
+
+    @on(Button.Pressed, button="#quit")  # (3)!
+    def quit(self):
+        """Called when the quit button is pressed."""
+        self.exit()
+
+    @on(Button.Pressed, super_cool_button="#button")  # (4)!
```

Comment:
We can't have examples that are simply dead code. Examples should work out of the box. No "exercise left to the reader".

Reply:
This is showing what happens if you provide `kwargs` for attributes that don’t exist. The example works out of the box, it’s just that this method will never be called.

Reply:
Yes, but it doesn't do anything. Examples should show the user what to do, not what *not* to do.